lib/sync-settings.coffee

```diff
@@ -300,6 +302,64 @@ SyncSettings =
         console.debug "config.set #{keyPath[1...]}=#{value}"
         atom.config.set keyPath[1...], value
 
+  removeRedundantPackages: (packages, cb) ->
```

Comment:
Can you please rename the function to `removeObsoletePackages`. The argument `packages` is kind of ambiguous - if it refers to the redundant / obsolete packages. Therefore I would suggest renaming the argument to `used_packages` (or does anyone have a better proposal?) to clarify which package names it contains.